src/utils/fileSystemHandler.ts

```diff
@@ -19,6 +19,9 @@ export function ensureDirectoryExists(filePath: string): void {
 
 export function ensureFileExists(filePath: string): void {
   ensureDirectoryExists(path.dirname(filePath));
+  if (fs.existsSync(filePath)) {
+    fs.unlinkSync(filePath);
+  }
```

Comment:
I'm not too concerned with performance degradation from this but I'm very curious if this only happens with the Task field label issue or if it's more prevalent.

Reply:
As far as I know it is at least Task and Idea. I have also tried `fs.truncate` but without success.